frontend/.storybook/preview.js

```diff
@@ -9,6 +10,18 @@ export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
 }
 
+const currentUrl = window.location.href;
+const isLocalhost = currentUrl.startsWith("http://localhost:6006/");
+const mockServiceWorkerUrl = isLocalhost ? "mockServiceWorker.js" : "https://" + window.location.hostname + "/mockServiceWorker.js";
```

Comment:
Im skeptical whether it is best practice to put ternary in a const var

Reply:
We discussed, and decided to leave this as it is.